packages/math-input/src/components/keypad/keypad.tsx

```diff
@@ -82,12 +78,11 @@ export default function Keypad(props: Props) {
     const [isMounted, setIsMounted] = React.useState<boolean>(false);
 
     // We don't want any tabs available on mobile fractions keypad
-    const availableTabs = getAvailableTabs(props);
+    const availableTabs = getAvailableTabs({...props, extraKeys});
```

Comment:
In this case, `props` doesn't include `extraKeys` because we deconstructed `Props` in the function definition (`{extraKeys = [], ...props}`).  So `props` contains everything but `extraKeys` and we have to add it back in to call `getAvailableTabs`.  I couldn't think of a cleaner way when adding the default value.